src/core/lib/experiments/experiments.h

```diff
@@ -25,27 +25,61 @@
 
 namespace grpc_core {
 
-inline bool IsTcpFrameSizeTuningEnabled() { return IsExperimentEnabled(0); }
-inline bool IsTcpReadChunksEnabled() { return IsExperimentEnabled(1); }
-inline bool IsTcpRcvLowatEnabled() { return IsExperimentEnabled(2); }
-inline bool IsPeerStateBasedFramingEnabled() { return IsExperimentEnabled(3); }
-inline bool IsFlowControlFixesEnabled() { return IsExperimentEnabled(4); }
+enum ExperimentNames {
+  TcpFrameSizeTuning = 0,
```

Comment:
drop the `= X` -- first one is always implicitly 0, and dropping it will avoid merge conflicts

Reply:
(we could add a static assert for the first/last if we wanted to prove that)